spread.yaml

```diff
@@ -347,6 +347,9 @@ prepare: |
         rmdir "$DELTA_PREFIX"
     fi
 
+    # Take the MATCH function from spread and allow our shell scripts to use it.
+    type MATCH | tail -n +2 > $TESTSLIB/spread-funcs.sh
```

Comment:
isn't this redundant with including the same spread-funcs.sh below?

Reply:
Hmm? I must be missing something. The point of this line is to ensure we pick up the latest definition of MATCH from spread.

Reply:
then why ship spread-funcs.sh at all?

Reply:
maybe i should've read the comment on the file before asking that; you answer my question perfectly. Thank you.

Reply:
Because you can use it for local testing and spellcheck that it's all fine.

Reply:
Why the tail -n +2 here ? What is in the frist lines?

Reply:
The +2 syntax means 2nd and all the following lines.